focuspoints.lrdevplugin/AppleDelegates.lua

```diff
@@ -0,0 +1,70 @@
+--[[
+  Copyright 2017 philmoz & Joshua Musselwhite, Whizzbang Inc
```

Comment:
Due to my lack of lawyer'ign and from what I'm seeing, I'm going to request that the  license stays as-is at this time. Legals suggest a Contributor License Agreement....which I see nobody else on github using.